assets/scss/modules/_buttons.scss

```diff
@@ -78,6 +78,27 @@
       pointer-events: none;
       opacity: 0.4;
     }
+
+    &.button--has-icon {
+      width: 36px;
+      height: 36px;
+      border: 1px solid $dp-border-grey;
+      border-radius: 0;
+      font-size: 22px;
```

Comment:
dont have var for this value?

Reply:
Good catch, Fede! Pushing the changes right away 👍